packages/amplify-frontend-javascript/lib/frontend-config-creator.js

```diff
@@ -32,11 +32,34 @@ function createAmplifyConfig(context, amplifyResources) {
   }
 }
 
-function createAWSExports(context, amplifyResources) {
-  const { serviceResourceMapping } = amplifyResources;
+async function createAWSExports(context, amplifyResources, cloudAmplifyResources) {
+  const newAWSExports = getAWSExportsObject(amplifyResources);
+  const cloudAWSExports = getAWSExportsObject(cloudAmplifyResources);
+  const currentAWSExports = await getCurrentAWSExports(context);
+
+  const customConfigs = getCustomConfigs(cloudAWSExports, currentAWSExports);
+
+  Object.assign(newAWSExports, customConfigs);
+  generateAWSExportsFile(context, newAWSExports);
+  return context;
+}
+
+function getCustomConfigs(cloudAWSExports, currentAWSExports) {
```

Comment:
We should move common function to its own package. This would help people who wants to write custom frontends

Reply:
We discussed this offline and I think this is out of scope for this PR. We can discuss more abouot this as a team.